pkgs/applications/science/math/gap/default.nix

```diff
@@ -141,12 +154,7 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     description = "Computational discrete algebra system";
-    maintainers = with maintainers;
-    [
-      raskin
-      chrisjefferson
-      timokau
-    ];
+    maintainers = with maintainers; [ chrisjefferson ] ++ teams.sage.members;
```

Comment:
I just talked to @ChrisJefferson and I don't think he's using NixOS anymore, so perhaps remove him? But he can and should of course say so himself

Reply:
Removed, we can always add him back later if he so desires.

Reply:
I will add the comment suggested by 7c6f434c ("We are also grateful to ChrisJefferson for previous work on the package and help with GAP-related questions from the upstream point of view.") before pushing.